jest.config.js

```diff
@@ -1,6 +1,30 @@
-const neutrino = require('neutrino');
+process.env.REACT_APP_NODE_ENV =
+  process.env.REACT_APP_NODE_ENVNODE_ENV || 'test';
+process.env.REACT_APP_DASHBOARD = process.env.REACT_APP_DASHBOARD || 'awfy';
 
-process.env.NODE_ENV = process.env.NODE_ENV || 'test';
-process.env.DASHBOARD = process.env.DASHBOARD || 'awfy';
+module.exports = {
+  // The root of your source code, typically `/src`
+  roots: ['<rootDir>/src'],
+  collectCoverageFrom: ['src/**/*.{js,jsx}'],
+  coveragePathIgnorePatterns: ['_tests_', 'index'],
+  // Paths to modules that run some code to configure or set up the testing environment before each test
+  testMatch: ['<rootDir>/_tests_/**/*.{js,jsx}'],
```

Comment:
I'd move all tests to `__tests__` instead of `_tests_` for convention (this is the most common place for tests).
_and_ you need to configure the entry to accept only `.test.` files, because some files in _tests_ are not tests and trigger an error when run, so something like that should work:
```
testMatch:  ["**/*.test.{js,jsx}"]
```
or if you want a stricter entry 
```
testMatch:  ["<rootDir>/__tests__/**/*.test.{js,jsx}"]
```

Reply:
Yes, that was a typos, _tests_ .